src/kOS/Suffixed/VesselTarget.cs

```diff
@@ -470,6 +473,16 @@ private void InitializeSuffixes()
             return crew;
         }
 
+        public MessageQueueStructure GetMessages()
+        {
+            if (Shared.Vessel.id != Vessel.id)
+            {
+                throw new KOSException("You can only access the message queue of the current vessel");
```

Comment:
Should we maybe make some kind of `KOSCommunicationException` (or maybe vessel/processor specific versions) instead of using the raw `KOSException`?  Is there any real benefit other than formatting? (might really be a question for @erendrake )